gensrc/thrift/Data.thrift

```diff
@@ -92,5 +92,9 @@ struct TStatisticData {
     9: optional string max
     10: optional string min
     11: optional string histogram
+    // global dict for low cardinality string
+    12: optional binary dict
```

Comment:
Why not define a struct to describe this dict, in order to make it readable.

Reply:
What's the binary content?

Reply:
```
agg function: dict_merge
input_type: TYPE_ARRAY (VARCHAR)
return_type: TYPE_VARCHAR (binary)

|dict-code: 4 bytes|string-size 4 bytes|string-content|...
```

Reply:
Why not define the content in thrift format? For performance? It seems not a performance issue.

Reply:
The dict content is the aggregate function result. Why could be thrift format ???

Reply:
There are several benefits to using thrift.
1. clearer definitions and more intuitive for the reader.
2. better compatibility. In the future when the structure changes, it is easier to be backwards and forwards compatible.
3. Better programming interface. No additional serialisation and deserialisation procedures are needed, and the structure can be modified directly.
4. More cross-language friendly. It is easier to manipulate structures when they need to be manipulated across languages.

If nothing else is bad, I would prefer to define it as a thrift structure for future extensibility.